ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -52,6 +55,44 @@ var (
 	podUIDIndexField = "metadata.uid"
 )
 
+// getDiscoveryClient returns a discovery client for the current reconciler
+func getDiscoveryClient(config *rest.Config) (*discovery.DiscoveryClient, error) {
+	return discovery.NewDiscoveryClientForConfig(config)
+}
+
+// Check where we are running. We are trying to distinguish here whether
+// this is vanilla kubernetes cluster or OPenshift
+func getClusterType(logger logr.Logger) bool {
+	// The user can explicitely overwrite usage of Route when running on OpenShift
+	// In this case operator will create ingress even if running on OpenShift
```

Comment:
Perhaps this comment should read:
```suggestion
	// The user can explicitely overwrite the usage of Route when running on OpenShift.
	// In this case operator will create an ingress even if running on OpenShift.
```